### PR TITLE
fix(droid): TabView header not showing

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes https://github.com/unoplatform/ziidms-private/issues/31

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

TabView header not showing on Android.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml [hunk 1]
```diff
@@ -701,6 +701,12 @@
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
+                            <!-- Uno workaround: On Android, the Height of ContentPresenters inside this Grid is not bing calculated correctly -->
+                            <!-- It can be removed once this PR is merged: https://github.com/unoplatform/uno/pull/18261 -->
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
```
Comment (ajpinedam) [inline src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml:708]: I tried to reproduce this behavior outside of the TabView context inside an isolated test, but I was not able to do it.

File: src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml [hunk 2]
```diff
@@ -22960,6 +22960,10 @@
                 <ColumnDefinition Width="*" />
                 <ColumnDefinition Width="Auto" />
               </Grid.ColumnDefinitions>
+              <!-- Uno workaround: On Android, the ContentPresenters inside this are not calculating correctly -->
+              <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+              </Grid.RowDefinitions>
```
Comment (jeromelaban) [inline src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml:22966]: Let's scope to android using XAML conditions.

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml [hunk 3]
```diff
@@ -701,6 +701,12 @@
                                 <ColumnDefinition Width="Auto" />
                             </Grid.ColumnDefinitions>
 
+                            <!-- Uno workaround: On Android, the Height of ContentPresenters inside this Grid is not bing calculated correctly -->
+                            <!-- It can be removed once this PR is merged: https://github.com/unoplatform/uno/pull/18261 -->
+                            <Grid.RowDefinitions>
```
Comment (jeromelaban) [inline src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml:706]: Let's scope to android using XAML conditions.

Comment (ajpinedam) [reply src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/TabView.xaml]: Done!

File: src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml (hunk 2 above)
Comment (ajpinedam) [reply src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml]: Done!

**Review by MartinZikmund — APPROVED** [review]

**Comment by jeromelaban** [comment]
@ajpinedam https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=146283&view=logs&j=a54adc20-f97a-5e02-19e8-cc33821dfc3e&t=50c2e5c6-4861-5ee5-b4ae-73b4c495ee2b&l=115

**Comment by jeromelaban** [comment]
@ajpinedam could you take a look at the build errors: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=146383&view=logs&j=50eac85d-e4f1-52a9-e299-c16e8a5f721f&t=f3688660-31ca-56fb-ec9e-b4e953c28722&l=142?

**Comment by ajpinedam** [comment]
@jeromelaban  are we backporting this PR?

**Pull request merged**